src/plugins/list/list.c

```diff
@@ -306,21 +306,29 @@ static int runPlugins (KeySet * pluginKS, KeySet * modules, KeySet * plugins, Ke
 				KeySet * sysConfigAll = ksCut (config, sysConfCutPoint);
 				KeySet * userConfigAll = ksCut (config, userCutPoint);
 				KeySet * pluginConfig = ksCut (userConfigAll, current);
-				realPluginConfig = elektraRenameKeys (pluginConfig, "user");
+				// replace "user/plugins/#X" with "user/"
+				KeySet * pluginConfigWithConfigPrefix = elektraRenameKeys (pluginConfig, "user");
 				ksDel (pluginConfig);
-				Key * toRemove = keyNew ("user/plugins", 0);
+				// append system config to plugin?
+				// seems like it is not used anymore
+				// TODO remove?
+				/*Key * toRemove = keyNew ("user/plugins", 0);
 				ksDel (ksCut (sysConfigAll, toRemove));
-				ksAppend (realPluginConfig, sysConfigAll);
-				keyDel (toRemove);
-				toRemove = keyNew ("user/placements", 0);
-				ksDel (ksCut (realPluginConfig, toRemove));
-				ksRewind (realPluginConfig);
```

Comment:
This code seems unused and without function. I tried to use it by setting `system/elektra/globalplugins/system/plugins/#1/config/...` instead of `system/elektra/globalplugins/user/plugins/#1/config/...` but it seems like the backend did not pass it to the list plugin.
@markus2330 or @tom-wa can you remember what the use case for this was?

Reply:
Only @tom-wa was involved in this implementation. I just read his docu and implemented backend.cpp accordingly. See #1855

Reply:
@waht 
>can you remember what the use case for this was?

everything below `system` was passed as system config to all plugins, while `user/plugins/#1/config` was only passed to plugin `#1`

Reply:
Ok I see. I changed it so that common plugin config can be passed using "/config" and added documentation for it. It should be usable now when mounting manually as well.